index.js

```diff
@@ -40,7 +40,8 @@ function DB (opts) {
     map: function (row, next) {
       if (!row.value) return null
       var v = row.value.v, d = row.value.d
-      if (v && v.lat !== undefined && v.lon !== undefined) {
+      var k = row.value.k
+      if (k && v.lat !== undefined && v.lon !== undefined) {
```

Comment:
Given this is is performance-critical code I think better use a comment rather than function, which would have a slight performance overhead.

One thing though, a delete could have lat and lon defined, that wouldn't affect here right?